apps/sharedtest/test/unit/dialer/font_size_manager_test.js

```diff
@@ -211,6 +225,34 @@ suite('font size manager', function() {
         });
       });
 
+      suite('on child <bdi>', function() {
+        setup(function() {
+          originalInnerHTML = document.body.innerHTML;
+          document.documentElement.style.fontSize = ROOT_FONT_SIZE + 'px';
+
+          view = document.createElement('div');
+          var bdiNode = document.createElement('bdi');
```

Comment:
We shouldn't really be creating ```var```'s here. Let's move this declaration up to where ```view``` is declared.